File: api/Proyecto_BK/Proyecto_BK.Common/Models/MunicipioViewModel.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations.Schema;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Proyecto_BK.Common.Models
{
    public class MunicipioViewModel
    {

        public string Muni_Codigo { get; set; }
        public string Muni_Descripcion { get; set; }
        public string Dept_Codigo { get; set; }
        public int Muni_Usua_Creacion { get; set; }
        public DateTime Muni_Fecha_Creacion { get; set; }
        public int? Muni_Usua_Modifica { get; set; }
        public DateTime? Muni_Fecha_Modifica { get; set; }

        [NotMapped]
        public string Dept_Descripcion { get; set; }
        [NotMapped]
        public string Usua_Creacion { get; set; }
        [NotMapped]
        public string Usua_Modifica { get; set; }
  
    }
}

File: app/plantilla2/src/app/demo/models/MunicipioViewModel.ts
export class Municipio {
    muni_Codigo?: string;
    muni_Descripcion?: string;
    dept_Codigo?: string;
}

export class MunicipioEnviar {
    Muni_Codigo: string;
    Muni_Descripcion: string;
    Dept_Codigo: string;
}

export class Fill {
    muni_Codigo: string;
    muni_Descripcion: string;
    dept_Codigo: string;
    muni_Usua_Creacion?: string;
    muni_Fecha_Creacion?: string;
    muni_Usua_Modifica?: string;
    muni_Fecha_Modifica?: string;
}

export class dropMunicipio{
    value?:String;
    text?:String;
    muni_Usua_Creacion?: number;
    muni_Fecha_Creacion?: Date;
    muni_Usua_Modifica?: number;
    muni_Fecha_Modifica?: Date;
}
export class DropMunicipios{
    muni_Codigo?: string;
    muni_Descripcion?: string;
}
